### ISSUE TITLE
Project website is unreachable via HTTPS

### ISSUE THREAD

**Issue by DebexeL** [Guake/guake#2043]
Hi.

Sorry if this is the wrong place to report this, but I didn't find any other way to bring this to your attention.

The project website (http://guake-project.org/) is unreachable via HTTPS.
This is an inconvenience (mild, but still an inconvenience) to users (like me) who have addons in place to force sites to use HTTPS.

Would you consider fixing it so the site is accessible via HTTPS?

Thank you. :)

**Comment by Davidy22** [comment]
Pinging @gsemet since they have the website

**Comment by gsemet** [comment]
Hi. I see what I can do. I use the lowest tier from gandi (still payed) but it does not seem to provide https certificates for free.

**Comment by DebexeL** [comment]
Pardon me for saying this, but if a webhosting company requires you to purchase a better tier to allow SSL certificates, then they are just after the profit. There's no reason for them to charge for extra for a certificate that is free (yes, it is free even for them). I would change away from such provider if I was their client. :D

**Comment by gsemet** [comment]
I agree. Generating the certificate is free but setting up into there http(s) server does not seem to be possible with this tier. That's a shame.
